docs/assets/documents/governance-for-careCommunication.md

```diff
@@ -89,8 +89,8 @@ It is optional for the system to support forwarding of a CareCommunication; howe
 
 |Rule name|Rules to contrain the use of forwarding CareCommunications|
 |:---|:---|
-|medcom-carecommunication-13 | User of sender system SHOULD be able to forward all CareCommunication in a message thread, as illustrated in <a href="#Fig3">Figure 3</a>. In this case, the forwarding **SHALL** include all previous message segments and the sender system **SHALL** create a new message thread with a new, unique communication identifier.|
-|medcom-carecommunication-14 | User of sender system SHOULD be able to forward the latest sent or received CareCommunication or a previously sent or received CareCommunication. If the user forwards and choose specific parts of the message, only the selected CareCommunication **SHALL** be included. |
+|medcom-carecommunication-13 | User of sender system **SHOULD** be able to forward all CareCommunication in a message thread, as illustrated in <a href="#Fig3">Figure 3</a>. In this case, the forwarding **SHALL** include all previous message segments and the sender system **SHALL** create a new message thread with a new, unique communication identifier.|
+|medcom-carecommunication-14 | User of sender system **SHOULD** be able to forward the latest sent or received CareCommunication or a previously sent or received CareCommunication. If the user forwards and choose specific parts of the message, only the selected parts **SHALL** be included, meaning the associated message segment(s) and Provenances. It is not allowed only to removed attachment, as this will cause the references to the payload in the Provenance to be incorrect.|
```

Comment:
It is not allowed only to removed attachment,
should be
It is not allowed only to remove attachment(s),